stdlib/internal/debug/debug_test.flux

```diff
@@ -44,3 +44,13 @@ testcase sink {
 
     testing.diff(got, want) |> yield()
 }
+
+testcase get_option {
+    got = debug.getOption(pkg: "internal/debug", name: "vectorize")
+    want = false
+
+    testing.diff(
+            got: array.from(rows: [{ v: got }]),
+            want: array.from(rows: [{ v: want }])
+        ) |> yield()
+}
```

Comment:
Can you add some more tests here, with the option set to true? And also test that the option is visible inside of the `compile` package, by calling `getOption` from inside of `map`.

Example:
```js
option debug.vectorize = true

...

got = array.from(rows: [{}]) |> map(fn: (r) => ({getOption: getOption(pkg: "internal/debug", name: "vectorize")}))

```

Reply:
fixed, though `array.from(rows: [{}])` doesn't actually create any rows over which to iterate over :/ . Couldn't see an easy way to fix it to emit an empty row either.

```
package array_test

import "array"
import "testing"

testcase from_empty_record {
    want = array.from(rows: [{ v: 123 }])
    got = array.from(rows: [{}]) |> map(fn: (r) => ({ v: 123 }))

    testing.diff(got, want) |> yield()
}
```